### DISCUSSION TITLE
Prompt-Free-Diffusion integration (seecoder)

### DISCUSSION THREAD

**Discussion by ledadu** [comfyanonymous/ComfyUI#876] [Ideas]
Can you integrate a Prompt-Free Diffusion : https://github.com/SHI-Labs/Prompt-Free-Diffusion

A extension for automatic111 already exist :
https://github.com/xingqian2018/sd-webui-prompt-free-diffusion

Maybe just custom node can be done

**Comment by ltdrdata** [comment]
Check this. 

https://github.com/BlenderNeko/ComfyUI_SeeCoder

This is implemented earlier.
